src/fonduer/candidates/mentions.py

```diff
@@ -27,13 +30,16 @@ def apply(self, x):
 
 class Ngrams(MentionSpace):
     """
-    Defines the space of Mentions as all n-grams (n_min <= n <= n_max) in a Sentence
-    _x_, indexing by **character offset**.
+    Defines the space of Mentions as all n-grams (n_min <= n <= n_max) in a
+    Sentence *x*, indexing by **character offset**.
 
     :param n_min: Lower limit for the generated n_grams.
+    :type n_min: int
     :param n_max: Upper limit for the generated n_grams.
-    :param split_tokens: Tokens, on which unigrams are split into two separate unigrams.
-
+    :type n_max: int
+    :param split_tokens: Tokens, on which unigrams are split into two separate
```

Comment:
This should probably also indicate that unigrams are split exclusively on the FIRST occurrence of a split token.

Reply:
Agreed.